docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -92,6 +92,7 @@ Now let's make a new project using our versioning system. Just type:
 
     mkdir myproject.git
     git init --bare myproject.git
+    mkdir myproject
```

Comment:
Is the idea that the `myproject.git` directory is inside `myproject`? If so, this line should go at the beginning and there should be a `cd myproject` line after it. Or am I misunderstanding?